docs/source/best-practices/authentication.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Authentication and Authorization
```

Comment:
Should authorization be in here? Also, why mutations specifically? Maybe it's too early to review this article ATM

Reply:
I think authorization fits in nicely for now. If we expand on the different ways to authenticate, then we might split it up